### PR TITLE
Update slogans.json

### PR DESCRIPTION
**Pull request by hey-shrey**

### CODE REVIEW

File: slogans.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   "2020-07-30": { "slogan": "just bake it", "via": "@_vramana" },
   "2020-07-31": { "slogan": "just hodl it", "via": "@_vramana" },
   "2020-08-01": { "slogan": "just bake it", "via": "@_vramana" },
-  "2020-08-02": { "slogan": "just _____ it", "via": "@your-twitter-handle" },
-  "2020-08-03": { "slogan": "just _____ it", "via": "@your-twitter-handle" },
+  "2020-08-02": { "slogan": "just brew it", "via": "@backslash_r" },
```
Comment (siddharthkp) [inline slogans.json]: duplicate entry 😅

**Review by siddharthkp — CHANGES_REQUESTED** [review]

**Comment by siddharthkp** [comment]
especially love just vue it 😋

**Comment by hey-shrey** [comment]
I'm worried multiple PR's might introduce conflicts with the date.

**Comment by siddharthkp** [comment]
don't worry about it :) I'm making sure of that when i merge

**Pull request merged**